src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -116,6 +119,11 @@ public static LoanPolicy unknown(String id) {
           errors.add(loanPolicyValidationError(CAN_NOT_RENEW_ITEM_ERROR));
           return failedValidation(errors);
         }
+        if (isFixed(getLoansPolicy()) && hasAlternateRenewalLoanPeriodForHolds()) {
```

Comment:
If possible, I think this should not be an error, rather it should be a server error, as the client cannot change their request in order to resolve this, as it relates to other configuration in the server.

What do you think?

Reply:
make sense.

Reply:
done